client/client.go

```diff
@@ -237,6 +269,18 @@ func getApiResp(r io.Reader) (ApiResponse, error) {
 		return resp, err
 	}
 	resp.Success = apiResp.Authorized
+
+	resp.usageReports = make(UsageReports, len(apiResp.UsageReports.Reports))
+	for _, report := range apiResp.UsageReports.Reports {
+		converted, err := report.convert()
+		if err != nil {
+			//TODO - what should we do here? Accept a logger into the client perhaps. Don't want the client to panic
+			// Situation is unlikely to occur so perhaps ignore?
```

Comment:
Just leaving the reports empty might be fine for now - further down the road some sort of logging mechanism and/or mechanism for signalling these errors might be interesting to explore.

Reply:
Cool will update the comment then. Cheers